### DISCUSSION TITLE
v1.10.0 RC

### DISCUSSION THREAD

**Discussion by ItzCrazyKns** [ItzCrazyKns/Vane#485] [Announcements]
## What's new
- Added file upload functionality in webSearch focus mode. Currently, if any other search mode is selected other than web search, it will automatically change it to web search (As implementing file uploads in other focus modes is left, it will be added by the meta search agent class which will be introduced in the next version)
- Added `keep_alive` variable to specify the time for how long the model should be loaded in Ollama (Defaults to '5m')

<hr /><em>This discussion was created from the release <a href='https://github.com/ItzCrazyKns/Perplexica/releases/tag/v1.10.0-rc'>v1.10.0 RC</a>.</em>

**Comment by datascience279** [comment]
I cannot see the file uploading option @ItzCrazyKns can you please help?

**Reply by ItzCrazyKns** [reply]
Follow the update guide https://github.com/ItzCrazyKns/Perplexica/blob/master/docs/installation/UPDATING.md

**Reply by datascience279** [reply]
Thanks!
@ItzCrazyKns as of now what type of files are supported? I see PDF works well but it keeps on loading on Excel and Word

**Reply by ItzCrazyKns** [reply]
Excel files are not supported ATM. You can only select the files selectable in the file input (I've specified all the types there so it won't allow you to load other types)

**Reply by datascience279** [reply]
Thank you!
